DataOnion/AuthExtensions.cs

```diff
@@ -75,13 +86,31 @@ public IFluentAuthStategyOnion ConfigureRedis(string connectionString)
             return this;
         }
 
-        public IFluentRedisOnion AddCustomAuthStrategy<T>(IAuthServiceStrategy<T> authStrategy)
+        public IFluentAuthOnion AddCustomAuthStrategy<T>(IAuthServiceStrategy<T> authStrategy)
         {
             _serviceCollection.AddScoped(typeof(IAuthService<>), typeof(AuthService<>));
             _serviceCollection.AddSingleton<IAuthServiceStrategy<T>>(authStrategy);
 
             return this;
         }
+
+        public IFluentAuthOnion ConfigureTwoFactorAuth(
+            string envPrefix,
+            string twoFactorAuthPrefix
+        )
+        {
+            _serviceCollection.AddScoped<ITwoFactorRedisContext, TwoFactorRedisContext>( provider =>
+                new TwoFactorRedisContext(
+                    envPrefix,
+                    twoFactorAuthPrefix,
+                    provider.GetService<IRedisManager>(),
```

Comment:
Non-optional services should use `GetRequiredService`

Reply:
Updated this